.travis.yml

```diff
@@ -24,4 +24,6 @@ install:
   - pip install .
 script:
   - flake8 . --exclude=.venv --ignore=E501
-  - nosetests
+  # run nose with doctests
+  # excludes files with missing or broken dependencies
+  - nosetests --with-doctest -I selector.py -I embedding.py -I obabel.py
```

Comment:
It would be better to have a separate file, a white list of files that we want to test.
We should start growing this white list over the next weeks and finally switch them over to a black list, with the final goal to test all files.